testsuite/run_sets/testsuite.yml

```diff
@@ -43,6 +43,7 @@
 
 # IDEMPOTENT
 
+- features/srv_change_task_schedule.feature
```

Comment:
maybe not as the first feature ;-) .

move it below, next to some other task-releated feature